Unix/server/server.c

```diff
@@ -1047,6 +1055,25 @@ int servermain(int argc, const char* argv[])
 
 #endif
 
+    /* If ntlm cred file is in use, check permissions and set NTLM_USER_FILE env variable */
+
+    char *ntlm_user_file = getenv("NTLM_USER_FILE");
```

Comment:
This case is easy since shell has much for flexible parsing for environment variables ...